medicines/web/pages/about.tsx

```diff
@@ -15,10 +15,6 @@ const StyledMain = styled.main`
   a {
     color: ${mhra70};
   }
-
-  ul li {
```

Comment:
this is deliberate, per https://app.zeplin.io/project/5dd51ae21205c944f8c1d35b/screen/5dea7c677a62584b2d341204